cmd/proxy-server/main.go

```diff
@@ -598,17 +599,19 @@ func refreshAdminTokenHandler(log *logrus.Entry) http.Handler {
 			JWTSigningSecret: JWTSigningSecret,
 		})
 		if err != nil {
-			log.WithError(err).Error("refreshing admin token")
-			http.Error(w, err.Error(), http.StatusInternalServerError)
+			if err := web.JSONErrorResponse(w, http.StatusInternalServerError, fmt.Errorf("refreshing admin token")); err != nil {
```

Comment:
The error can be included here:

fmt.Errorf(""refreshing admin token: %v", err)

Reply:
fixed